data-shaping/converter/data_model_mapping.py

```diff
@@ -0,0 +1,131 @@
+SIMPLE_REFERENCE_MAPPING = {
+    "TraineeStatus": {
+        "id": {"sheet": "tlkpStatus", "column": "STID"},
+        "title": {"sheet": "tlkpStatus", "column": "STSTAT"},
+    },
+    "EmailPreference": {
+        "id": {"sheet": "tlkpEmailPref", "column": "ETID"},
+        "title": {"sheet": "tlkpEmailPref", "column": "ETPREF"},
+    },
+    "TrainingProgram": {
+        "id": {"sheet": "tlkpProgramType", "column": "PRID"},
+        "title": {"sheet": "tlkpProgramType", "column": "PRPRGM"},
+    },
+    "Salary": {
+        "id": {"sheet": "tlkpSalary", "column": "SRID"},
+        "title": {"sheet": "tlkpSalary", "column": "SRSALARY"},
+    },
+    "JobSector": {
+        "id": {"sheet": "tlkpFirstJobType", "column": "FJID"},
+        "name": {"sheet": "tlkpFirstJobType", "column": "FJTYPE"},
+    },
+    "NHSBand": {
+        "id": {"sheet": "tlkpAFCBand", "column": "BDID"},
+        "title": {"sheet": "tlkpAFCBand", "column": "BDBAND"},
+    },
+    "ContractType": {
+        "id": {"sheet": "tlkpContractType", "column": "COID"},
+        "title": {"sheet": "tlkpContractType", "column": "COTYPE"},
+    },
+    "Title": {
+        "id": {"sheet": "tlkpTitle", "column": "id"},
+        "title": {"sheet": "tlkpTitle", "column": "title"},
+    },
+    "Gender": {
+        "id": {"sheet": "tlkpGender", "column": "GNID"},
+        "title": {"sheet": "tlkpGender", "column": "GNGNDR"},
+    },
+    "Ethnicity": {
+        "id": {"sheet": "tlkpEthnic", "column": "ECID"},
+        "title": {"sheet": "tlkpEthnic", "column": "ECDESC"},
+    },
+    "DisabilityCategory": {
+        "id": {"sheet": "tlkpDisabilityCategory", "column": "DCID"},
+        "title": {"sheet": "tlkpDisabilityCategory", "column": "DCDESC"},
+    },
+    "Religion": {
+        "id": {"sheet": "tlkpReligion", "column": "RLID"},
+        "title": {"sheet": "tlkpReligion", "column": "RLREL"},
+    },
+    "SexualOrientation": {
+        "id": {"sheet": "tlkpSexor", "column": "SOID"},
+        "title": {"sheet": "tlkpSexor", "column": "SODESC"},
+    },
+    "Nationality": {
```

Comment:
This outputs as blank, it does look like that it wasn't supplied, but just a flag if we have different versions of the original files.

Reply:
Cheers! We can look at it in testing